App/Lettuce/Sources/Lettuce/MessageCenter/ListMessages/GET/ListMessagesInThreadRequest.swift

```diff
@@ -0,0 +1,59 @@
+// system
+import Foundation
+import HTTPTypes
+
+// local
+import Onion
+
+struct ListMessagesInThreadRequest: Request {
+    typealias Output = [Message]
+    
+    var path: String {
+        return preparePathWithComponents()
+    }
+    
+   func preparePathWithComponents() -> String {
+        var components = URLComponents()
+        components.path = self.path
+        components.queryItems = []
+        
+        if let limit = limit {
+            components.queryItems?.append(URLQueryItem(name: "limit", value: String(limit)))
+        }
+        
+        if let offset = offset {
```

Comment:
There is this package https://github.com/sloik/SweetURL/blob/main/Sources/SweetURL/URLExtensions.swift#L59 with method `queryValues`. Could you extension method (maybe to `URLComponents?`) so this call site would look less dense?

Reply:
Fixed